compiler/rustc_middle/src/ty/query.rs

```diff
@@ -246,17 +257,10 @@ macro_rules! define_callbacks {
             #[inline(always)]
             pub fn $name(self, key: query_helper_param_ty!($($K)*)) -> query_stored::$name<'tcx>
             {
-                let key = key.into_query_param();
+                let key: $( $K )* = key.into_query_param();
```

Comment:
Why do we need to specify the type here and not in the `Ensure` case?

Reply:
No reason, I added it when trying to get this code to compile and forgot to remove it.